weed/storage/disk_location.go

```diff
@@ -23,9 +23,10 @@ type DiskLocation struct {
 	DiskType               types.DiskType
 	MaxVolumeCount         int
 	OriginalMaxVolumeCount int
-	MinFreeSpacePercent    float32
-	volumes                map[needle.VolumeId]*Volume
-	volumesLock            sync.RWMutex
+	// MinFreeSpace limits the minimum free space (<=100 as percentage, > 100 as bytes)
+	MinFreeSpace float32
```

Comment:
This is confusing. 

How about creating a type for this, e.g. MinimumFreeSpaceType, and put these logic inside, such as EmptyTo, ParseMinFreeSpace, etc.

Reply:
good idea

Reply:
code has refactored to create a MinFreeSpace type to separate relative logics like parsing and judging.